tables/atom.py

```diff
@@ -306,20 +306,26 @@ def from_sctype(cls, sctype, shape=(), dflt=None):
 
             >>> import numpy as np
             >>> Atom.from_sctype(np.int16, shape=(2, 2))
-            Int16Atom(shape=(2, 2), dflt=0)
+            Int16Atom(shape=(np.int64(2), np.int64(2)), dflt=np.int16(0))
```

Comment:
I'm not sure what to do about these reprs, in the tests I can use `f"{np.int64(2)!r}"` but in the docstrings I have to use either the numpy1 or numpy2 docstrings

I could skip these docstring tests on numpy 2 builds?

Reply:
I hacked these in to be 1/2 compatible by using `repr(x) == f" ... "` a bit horrible but it passes on both now

Reply:
I'm not running the doctests on numpy v2 anymore